dds/DCPS/ConfigStoreImpl.h

```diff
@@ -0,0 +1,148 @@
+/*
+ * Distributed under the OpenDDS License.
+ * See: http://www.opendds.org/license.html
+ */
+
+#ifndef OPENDDS_DCPS_CONFIG_STORE_IMPL_H
+#define OPENDDS_DCPS_CONFIG_STORE_IMPL_H
+
+#include "dcps_export.h"
+
```

Comment:
```suggestion
```

Export isn't a special header in terms of order. (In fact I could argue it should always go at the bottom to flush out places where we don't include it, but still use the export macro. I've found a couple of those cases before.)